user/sorghum-lines-genomics-year-2.md

```diff
@@ -0,0 +1,196 @@
+# Sorghum Lines for Genomics Experiment \(Year 2\)
+
+| **accession** | [**taxid**](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=4558) | [**organism common name**](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=4558) | **subspecific genetic lineage rank** | **subspecific genetic lineage name** | [**ploidy**](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor) | [**number of replicons**](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor) | [**estimated size**](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor) | [**source material identifiers**](https://www.genesys-pgr.org/) | **sample name** |
```

Comment:
The accession column is a unique identifier from the GRIN database. That works here at least since these are all from GRIN. I don't think we have any (or most) of the BAP in the cultivars table yet.